### PR TITLE
Making etcd available to compute pod

### PR DESCRIPTION
**Pull request by tlitovsk**
Signed-off-by: Tolik Litovsky tlitovsk@redhat.com

### CODE REVIEW

**Comment by tlitovsk** [comment]
The ci is strange and have some errors

File: specs/service-controller.json [hunk 1]
```diff
@@ -16,6 +16,11 @@
         "name": "controller-rest",
         "port": 8080,
         "targetPort":"controller-rest"
+      },
+      {
+        "name": "etc-rest",
```
Comment (fabiand) [inline specs/service-controller.json]: Any reason why this is called etc-rest and above etcd-rest?

Comment (tlitovsk) [inline specs/service-controller.json]: not really name is basically meaningnless in this case 
but will change

File: container/controller/example-dom [hunk 2]
```diff
@@ -47,9 +47,6 @@
       <address type='pci' domain='0x0000' bus='0x00' slot='0x06' function='0x2'/>
     </controller>
     <controller type='pci' index='0' model='pci-root'/>
-    <interface type='bridge'>
-      <source dev='br0' />
-    </interface>
```
Comment (fabiand) [inline container/controller/example-dom:52]: This will disconnect the Vm from the network. What is your plan to add networking again?

Comment (tlitovsk) [inline container/controller/example-dom:52]: My plan to make it working for the POC and worry about this later :-)

**Pull request merged**